zk/datadog_checks/zk/zk.py

```diff
@@ -59,14 +59,18 @@
 # stdlib
 from collections import defaultdict
 from distutils.version import LooseVersion  # pylint: disable=E0611,E0401
-from StringIO import StringIO
+from six import PY3, iteritems
 import re
 import socket
 import struct
+from six import StringIO
 
-# project
+from datadog_checks.dev.utils import ensure_bytes, ensure_unicode
```

Comment:
we don't ship the dev package, you should use the base check instead

Reply:
Ahh, nice catch!